cmake/dependencies/AMReX.cmake

```diff
@@ -279,7 +279,7 @@ set(WarpX_amrex_src ""
 set(WarpX_amrex_repo "https://github.com/AMReX-Codes/amrex.git"
     CACHE STRING
     "Repository URI to pull and build AMReX from if(WarpX_amrex_internal)")
-set(WarpX_amrex_branch "24.10"
+set(WarpX_amrex_branch "6239d2576489a3178d753471f988908f5499577c"
```

Comment:
I just merged https://github.com/AMReX-Codes/amrex/pull/4181

Do you mind running the AMReX update script once more so this PR then unblocks #5364 ? :pray: :)

Reply:
Done in [8c95534](https://github.com/ECP-WarpX/WarpX/pull/5369/commits/8c955348466ed5120dd150794e0058539cb0fc32).

Reply:
Thank you :)